File: articles/documentation-model-cards-and-datasheets-for-algorithms/c/documentation_quality_score.c
#include <stdio.h>

int main(void) {
    double scores[] = {0.62, 0.6875, 0.58, 0.50, 0.56, 0.52};
    double total = 0.0;
    for (int i = 0; i < 6; i++) total += scores[i];
    printf("documentation_quality_score=%.4f\n", total / 6.0);
    return 0;
}

File: articles/documentation-model-cards-and-datasheets-for-algorithms/rust/documentation_quality_score.rs
fn main() {
    let scores = [0.62_f64, 0.6875_f64, 0.58_f64, 0.50_f64, 0.56_f64, 0.52_f64];
    let total: f64 = scores.iter().sum();
    let quality = total / scores.len() as f64;
    println!("documentation_quality_score={:.4}", quality);
}
